docs(ui): update landing page design

matthewvogas opened the pull request:
### Summary

Index Page (Landing Page) Updated

- Laptop, tablet & mobile mode (Components and Page)
- Dark/Light mode
- Applied colors and styles from old Figma to new one

Figma link: https://www.figma.com/design/39MGGVIGQ1bL3FgVzuXEdC/Holdex?node-id=7355-63376&t=gg8CmTOHvTA9KGyC-4

Issue: https://github.com/holdex/marketing/issues/72 
Goal: https://github.com/holdex/marketing/issues/68

eta: 16h

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit


## Summary by CodeRabbit

- **Documentation**
	- Introduced a new "Update" section in the DESIGN.md file, linking to a Figma design document for improved access to design resources.

<!-- end of auto-generated comment: release notes by coderabbit.ai -->

angelicawill reviewed (commented):
Looks good overall! Left some comments in Figma. Don't forget to move the frame to our existing Figma and update the link in our repo.

markholdex reviewed (commented):
@matthewvogas please adjust the PR title. It's not following the guidelines. 

For designs we use docs prefix like in other examples. Also no need for `!` exclamation in it.

angelicawill commented:
@matthewvogas FYI: [naming guidelines](https://github.com/holdex/developers/blob/main/.github/CONTRIBUTING.md#naming) 

the PR name has to be from the POV of user

markholdex commented:
@matthewvogas I provided access to our Original Figma. Please transfer your work, make the necessary changes to the designs and update the link of your PR. Thx 🙏

matthewvogas commented:
@markholdex  thanks, I'll doing some changes in the design about your comments. Some styles are not connected well to Figma variables, so I will fix that as well. As soon as the design is ready, I send the PR of the design. I will adjust the ETA.

matthewvogas commented:
@markholdex I updated the light/dark version of the design, and connected everything (titles, colors, and components) to the new Figma with the style guides. I updated the link in the PR too.

markholdex commented on DESIGN.md:
```diff
@@ -1,3 +1,6 @@
+## Update
+- [/index](https://www.figma.com/design/ouzfjob16mSlDSCY7MT9ZM/Holdex-(https%3A%2F%2Fgithub.com%2Fholdex%2Fmarketing%2Fissues%2F68)?node-id=9032-65456&t=KTk0bdIQIktfo4Jx-4)
```
@matthewvogas PR link still wrong

markholdex commented:
@matthewvogas what is the ETA to get this completed?

matthewvogas commented:
> @matthewvogas I provided access to our Original Figma. Please transfer your work, make the necessary changes to the designs and update the link of your PR. Thx 🙏

I just updated it, both with a commit, and in the PR description.

matthewvogas commented:
Mobile Design Done  @markholdex - https://www.figma.com/design/39MGGVIGQ1bL3FgVzuXEdC/Holdex?node-id=7382-66909&t=gg8CmTOHvTA9KGyC-4
